src/common/components/ui/ClusterEventsToolbar.tsx

```diff
@@ -179,7 +176,7 @@ const ClusterEventsToolbar: React.FC<ClustersListToolbarProps> = ({
 
   const onDeleteChip: ToolbarFilterProps['deleteChip'] = (type, chip) => {
     if (type) {
-      const id = chip['key'] || chip;
+      const id = (chip['key'] || chip) as string;
```

Comment:
do we have any guarantee either `chip.key` or `chip` will be defined or will be a string?

Reply:
yes, we do